src/main/java/prover/Prover.java

```diff
@@ -36,6 +41,7 @@ public static void main(String[] args) throws IOException {
     options.addOption(new Option("t", "test", false, "Run trivial test to verify setup"));
     options.addOption(new Option("p", "primary-size", true, "Size of primary input (1)"));
     options.addOption(new Option("o", "output", true, "Output file (proof.bin)"));
+    options.addOption(new Option("c", "curve", true, "Curve (bn254a*, bls12-377)"));
```

Comment:
Why the `*` in `bn254a*`?

Reply:
It signifies the default value.  I thought it was pretty common, but changed in case it's not clear.

Reply:
Ah I see. Thanks, I didn't realize that meant default value - happy both ways, just wanted to make sure it wasn't a typo.